.github/workflows/publish-nuget.yaml

```diff
@@ -19,8 +20,23 @@ jobs:
       - name: Get version number
         id: version
         uses: PaulHatch/semantic-version@v5.2.1
-        with:
-          version_format: "${major}.${minor}.${patch}-beta${increment}"
+
+      - name: Generate semver version number
+        id: generate-version
+        shell: powershell
+        run: |
+          if("${{ steps.version.outputs.is_tagged }}" -eq "false")
+          {
+            $assembly_version = "${{ steps.version.outputs.major }}.${{ steps.version.outputs.minor }}.${{ steps.version.outputs.patch }}.${{ steps.version.outputs.increment }}"
```

Comment:
It seems that this is the same in both cases, so it can be moved outside the `if`.

Reply:
Thanks for noting - you are right